test/SignerBondsUniswapV2.test.js

```diff
@@ -74,6 +74,181 @@ describe("SignerBondsUniswapV2", () => {
     })
   })
 
+  describe("setMaxAllowedPriceImpact", () => {
+    const maxAllowedPriceImpact = 10000
+
+    context("when caller is not the governance", () => {
+      it("should revert", async () => {
+        await expect(
+          signerBondsUniswapV2
+            .connect(thirdParty)
+            .setMaxAllowedPriceImpact(maxAllowedPriceImpact)
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when max allowed price impact too big", () => {
+      it("should revert", async () => {
+        await expect(
+          signerBondsUniswapV2
+            .connect(governance)
+            .setMaxAllowedPriceImpact(maxAllowedPriceImpact + 1)
+        ).to.be.revertedWith("Maximum value is 10000 basis points")
+      })
+    })
+
+    context("when governance did not set max allowed price impact", () => {
+      const defaultPriceImpact = 100
+      it("should return the default value", async () => {
+        expect(await signerBondsUniswapV2.maxAllowedPriceImpact()).to.be.equal(
+          defaultPriceImpact
+        )
+      })
+    })
+
+    context("when max allowed price impact is in correct range", () => {
+      const priceImpact = 500
+      beforeEach(async () => {
+        expect(
+          await signerBondsUniswapV2
+            .connect(governance)
+            .setMaxAllowedPriceImpact(priceImpact)
+        )
+      })
+
+      it("should set max allowed price impact", async () => {
+        expect(await signerBondsUniswapV2.maxAllowedPriceImpact()).to.be.equal(
+          priceImpact
+        )
+      })
+    })
+  })
+
+  describe("setSlippageTolerance", () => {
+    const maxBasisPoints = 10000
+
+    context("when caller is not the governance", () => {
+      it("should revert", async () => {
+        await expect(
+          signerBondsUniswapV2
+            .connect(thirdParty)
+            .setSlippageTolerance(maxBasisPoints)
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when slippage tolerance too big", () => {
```

Comment:
is too high?

Reply:
Fixed in https://github.com/keep-network/coverage-pools/pull/156/commits/7293ab108d709dce06e5db22230eb2d5ad52d647.